rqt_top/src/rqt_top/node_info.py

```diff
@@ -53,7 +52,7 @@ def get_node_info(self, node_name, skip_cache=False):
                     return p
                 except:
                     return False
-        except error:
+        except SocketError:
```

Comment:
Sorry, that I make you invest this amount of time :) 
I don't understand your point.
`from socket import error as SocketError` is outside of the try now, since it should work with both versions (at least it does under ipython and ipyhton3). Therefore it should be defined.